blockchain/pallets/minting/src/tests.rs

```diff
@@ -282,6 +287,63 @@ mod register_identity {
         });
     }
 
+    #[test]
+    fn first_call_to_register_for_minting_is_free() {
+        use frame_support::pallet_prelude::Pays;
+
+        new_test_ext().execute_with(|| {
+            register_id_account(42, 1);
+
+            // first call is free, actual_weight set to 0
```

Comment:
This is a stale comment. This is one of the stronger reasons to not add comments to code. For unit tests, the tests should describe what is happening. For production code, when I used to add comments, I now extract functions or variables so I can give something a name.

Reply:
I agree, removed the comments